crates/ruff_python_formatter/src/statement/suite.rs

```diff
@@ -218,18 +255,27 @@ const fn is_import_definition(stmt: &Stmt) -> bool {
     matches!(stmt, Stmt::Import(_) | Stmt::ImportFrom(_))
 }
 
-fn is_docstring(stmt: &Stmt) -> bool {
+/// Checks if the statement is a simple string that can be formatted as a docstring
+fn get_docstring<'a>(stmt: &'a Stmt, locator: &Locator) -> Option<&'a ExprConstant> {
     let Stmt::Expr(ast::StmtExpr { value, .. }) = stmt else {
-        return false;
+        return None;
     };
 
-    matches!(
-        value.as_ref(),
-        Expr::Constant(ast::ExprConstant {
-            value: Constant::Str(..),
-            ..
-        })
-    )
+    let Expr::Constant(constant) = value.as_ref() else {
+        return None;
+    };
+    if let ExprConstant {
+        value: Constant::Str(..),
+        range,
+        ..
+    } = constant
+    {
+        if is_implicit_concatenation(locator.slice(*range)) {
```

Comment:
Oh no xD... The speedup potential for adding a `implicit_concatenated` to `String` increases day by day

Reply:
also the confusion for developers would make it worthwhile :bow: